hudi-sync/hudi-hive-sync/src/test/java/org/apache/hudi/hive/functional/HiveSyncFunctionalTestSuite.java

```diff
@@ -25,7 +25,7 @@
 import org.junit.runner.RunWith;
 
 @RunWith(JUnitPlatform.class)
-@SelectPackages("org.apache.hudi.hive.functional")
+@SelectPackages("org.apache.hudi.hive")
```

Comment:
we should just make the classes to "functional" instead of changing this?

Reply:
my bad. guess this change may not be required . will fix it.